tests/partitioninfo.spec.coffee

```diff
@@ -69,3 +70,59 @@ describe 'Partitioninfo:', ->
 					type: 131
 				}
 			])
+
+	describe '.get() disk 2', ->
+
+		it 'should be able to get the first logical partition of the extended one', ->
+			promise = partitioninfo.get(DISK2_PATH, 5)
+			m.chai.expect(promise).to.eventually.become
+				offset: 3145728
+				size: 1048576
+				type: 131
+
+		it 'should be able to get the second logical partition of the extended one', ->
+			promise = partitioninfo.get(DISK2_PATH, 6)
+			m.chai.expect(promise).to.eventually.become
+				offset: 5242880
+				size: 1048576
+				type: 131
+
+		it 'should return an error when asked for a non existing partition', ->
+			promise = partitioninfo.get(DISK2_PATH, 10)
+			m.chai.expect(promise).to.be.rejectedWith('Partition not found: 10.')
+
+	describe '.getPartitions() disk 2', ->
+		it 'should list all partitions of a disk image', ->
+			promise = partitioninfo.getPartitions(DISK2_PATH)
+			m.chai.expect(promise).to.eventually.become([
```

Comment:
I didn't want to break the API so `getPartitions()` is still returning an array.
I'll add a note in the README that `getPartitions()[n - 1]` may not be equal to `get(n)`.
The other solution would be to make `getPartitions()` return an object.

I'll add the unit-test.